cratedb_toolkit/io/processor/kinesis_lambda.py

```diff
@@ -127,8 +127,7 @@ def handler(event, context):
             connection.execute(sa.text(operation.statement), parameters=operation.parameters)
 
             # Processing alternating CDC events requires write synchronization.
-            # FIXME: Needs proper table name quoting.
-            connection.execute(sa.text(f"REFRESH TABLE {CRATEDB_TABLE}"))
+            connection.execute(sa.text(f"REFRESH TABLE {cdc.quote_table_name(CRATEDB_TABLE)}"))
```

Comment:
See [quote_table_name](https://github.com/crate/commons-codec/blob/v0.0.13/src/commons_codec/transform/dynamodb.py#L61-L70). It is currently just a poor man's version until we can shuffle code better and more sustainably.